lyonjs/shortvid.io
# feat(app): sidebar testing

## 🤔 Why do you want to make those changes?

We cannot send the feature without testing.

## 🧑‍🔬 How did you make them?

I created the test file under the folder `__test__/app/` and completed the test. 

## 🧪 How to check them?

- Check the code
- Check the actions
